util/weekly_build/sites/acorn.sh

```diff
@@ -27,3 +27,4 @@ PACKAGES_TO_INSTALL="ufs-weather-model-env global-workflow-env gsi-env madis"
 PADDED_LENGTH=140
 TEST_UFSWM=ON
 BATCHACCOUNT=NCEPLIBS-DEV
+FIND_CMD="find"
```

Comment:
@AlexanderRichert-NOAA Please check if this is correct or if you can use an optimized version such as `lfs find` (see Blueback site config below). Thanks.

Reply:
This is fine for now, I'll be doing a batch of weekly build-related updates soon so I'll change it there if needed.